src/goose/console/server.clj

```diff
@@ -0,0 +1,42 @@
+(ns goose.console.server
+  (:require
+    [goose.brokers.redis.broker :as redis]
+    [goose.console.handler :as handler]
+    [ring.adapter.jetty :as jetty]
+    [ring.middleware.file :refer [wrap-file]]
+    [ring.middleware.resource :refer [wrap-resource]]
+    [ring.util.response :as response]
+    [goose.defaults :as d]))
+
+(defonce server (atom nil))
+
+(defn routes [{:keys [uri broker]}]
```

Comment:
question: I have not used any routing library like bidi/compojure for this, since a custom function was enough. Once more routes will be added, we can add library. I would suggest using [bidi](https://github.com/juxt/bidi/blob/master/project.clj) based on less dependencies compared to [compojure](https://github.com/weavejester/compojure/blob/master/project.clj). Let me know what you think @olttwa ?